sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -440,73 +456,71 @@ void stabilization_indi_rate_run(struct FloatRates rate_sp, bool in_flight)
     // incremental thrust
     for (i = 0; i < INDI_NUM_ACT; i++) {
       v_thrust +=
-        (stabilization_cmd[COMMAND_THRUST] - actuator_state_filt_vect[i]) * Bwls[3][i];
+        (stabilization_cmd[COMMAND_THRUST] - use_increment*actuator_state_filt_vect[i]) * Bwls[3][i];
     }
   }
 
   // The control objective in array format
-  indi_v[0] = (angular_accel_ref.p - angular_acceleration[0]);
-  indi_v[1] = (angular_accel_ref.q - angular_acceleration[1]);
-  indi_v[2] = (angular_accel_ref.r - angular_acceleration[2] + g2_times_du);
+  indi_v[0] = (angular_accel_ref.p - use_increment*angular_acceleration[0]);
+  indi_v[1] = (angular_accel_ref.q - use_increment*angular_acceleration[1]);
+  indi_v[2] = (angular_accel_ref.r - use_increment*angular_acceleration[2] + g2_times_du);
   indi_v[3] = v_thrust;
 
-  if (in_flight) {
 #if STABILIZATION_INDI_ALLOCATION_PSEUDO_INVERSE
-    // Calculate the increment for each actuator
-    for (i = 0; i < INDI_NUM_ACT; i++) {
-      indi_du[i] = (g1g2_pseudo_inv[i][0] * indi_v[0])
-        + (g1g2_pseudo_inv[i][1] * indi_v[1])
-        + (g1g2_pseudo_inv[i][2] * indi_v[2])
-        + (g1g2_pseudo_inv[i][3] * indi_v[3]);
-    }
+  // Calculate the increment for each actuator
+  for (i = 0; i < INDI_NUM_ACT; i++) {
+    indi_du[i] = (g1g2_pseudo_inv[i][0] * indi_v[0])
+      + (g1g2_pseudo_inv[i][1] * indi_v[1])
+      + (g1g2_pseudo_inv[i][2] * indi_v[2])
+      + (g1g2_pseudo_inv[i][3] * indi_v[3]);
+  }
 #else
-    // Calculate the min and max increments
-    for (i = 0; i < INDI_NUM_ACT; i++) {
-      du_min[i] = -MAX_PPRZ * act_is_servo[i] - actuator_state_filt_vect[i];
-      du_max[i] = MAX_PPRZ - actuator_state_filt_vect[i];
-      du_pref[i] = act_pref[i] - actuator_state_filt_vect[i];
+  // Calculate the min and max increments
+  for (i = 0; i < INDI_NUM_ACT; i++) {
+    du_min[i] = -MAX_PPRZ * act_is_servo[i] - use_increment*actuator_state_filt_vect[i];
+    du_max[i] = MAX_PPRZ - use_increment*actuator_state_filt_vect[i];
+    du_pref[i] = act_pref[i] - use_increment*actuator_state_filt_vect[i];
 
 #ifdef GUIDANCE_INDI_MIN_THROTTLE
-      float airspeed = stateGetAirspeed_f();
-      //limit minimum thrust ap can give
-      if (!act_is_servo[i]) {
-        if ((guidance_h.mode == GUIDANCE_H_MODE_HOVER) || (guidance_h.mode == GUIDANCE_H_MODE_NAV)) {
-          if (airspeed < 8.0) {
-            du_min[i] = GUIDANCE_INDI_MIN_THROTTLE - actuator_state_filt_vect[i];
-          } else {
-            du_min[i] = GUIDANCE_INDI_MIN_THROTTLE_FWD - actuator_state_filt_vect[i];
-          }
+    float airspeed = stateGetAirspeed_f();
+    //limit minimum thrust ap can give
+    if (!act_is_servo[i]) {
+      if ((guidance_h.mode == GUIDANCE_H_MODE_HOVER) || (guidance_h.mode == GUIDANCE_H_MODE_NAV)) {
+        if (airspeed < 8.0) {
```

Comment:
Magic number, a define would be better, reuse the AC define that are already in use like NOMINAL_AIRSPEED or MINIMUM_AIRSPEED